Sample Models/Chemistry & Physics/Ising.nlogo

```diff
@@ -245,46 +219,72 @@ In the default settings for the model, the temperature is set fairly low.  What
+At each iteration, we give a fixed number of spins a chance to flip.  There are other possible methods, for example, the "checkerboard" update rule: if you imagine the rectangular lattice as being made of alternating white and black squares like a checkerboard, then at each iteration first all the white squares update simultaneously, then all the black squares update simultaneously.  On a parallel computer with many processors, and a programming language which takes advantage of them, the checkerboard rule would run much much faster. Since NetLogo only simulates parallelism using a single processor, the checkerboard rule does not run dramatically faster. (Note that if you use the checkerboard rule, the grid of spins must have even dimensions.)
 
 ## NETLOGO FEATURES
 
-Because only a very small amount happens during each tick, this is a model where the speed slider is especially important, since it lets the user speed the model up by not updating the view every tick.
+This model makes 1000 attempted flips in every iteration of the `GO` loop. It codes this as `repeat 1000 [ update ]`. Note that this does not change the core Ising algorithm that does one attempted patch flip at a time.
+
+However, this approach does necessitate a departure from the usual NetLogo practice of calling `tick` at the end of the `GO` procedure. If we called `tick` at the end, then the tick counter would advance only once for every 1000 attempted flips and the Ising model measures the magnetization at every attempted flip. We therefore use `tick-advance` primitive to advance the NetLogo clock and sync it up with the traditional Ising model. Because the `tick-advance` primitive doesn't update the NetLogo plots, we have to explicitly call `update-plots`.
+
+This last change also leads to one more departure from conventional NetLogo practice. In the Magnetization plot, we would usually have the code `plot magnetization`. But that code would plot the magnetization every time the `update-plots` code was called, which would be only once in every 1000 attempted flips of the Ising model and the x-axis of the plot would not reflect true Ising time steps. To sync up the plot with the usual measure of time in the Ising model, we use the code `plotxy ticks magnetization`.
+
+A more direct translation of the core Ising algorithm to NetLogo would be as follows:
+
+    to go
+      ask one-of patches [ update ]
+      tick
+    end
+
+However, this more direct version would update the NetLogo view at every tick, which would be at every potential patch flip. Because the view updates happen much more slowly than the calculations, the direct approach would dramatically slow down the model in its normal speed slider state. To significantly speed up the direct version, one would need to set the speed slider to its maximum value, so that NetLogo would not update the view at every tick.
```

Comment:
relevant: "note that there is a separate possibility of having an additional setting that says whether at the default setting, frames may be skipped or not (in order to hit the frame rate target)" from https://github.com/NetLogo/NetLogo/issues/741